### ISSUE TITLE
Cannot delete customer V 1.0.15

### ISSUE THREAD

**Issue by PeopleInside** [uvdesk/community-skeleton#444]
# Bug report

A past resolved bug returns again in the new version of UVdesk.
Seems with this new version you are loosing a lot of fix made in all this months.

Maybe some bug will be no more find if you re created with this new version. 
I think you miss to insert fix in the master branch so all resolved issue on the past are again present.

On version V 1.0.15 I'm unable to delete customers.
https://github.com/uvdesk/community-skeleton/issues/419

**Comment by PeopleInside** [comment]
I'm restoring the old version using the backup as seems something is wrong with the upgrade.
Waiting your replies on the forum and here on many issues.

I loosed a lot of time by upgrading and try to fix issues than put again customization... you should have a better upgrade procedure and made an announcement with detailed steps or you made me loose a lot of time and doing a lot of not useful work. Now I will lost everything I done in this last hours and resume the old version.

**Comment by PeopleInside** [comment]
Also if I made:

```
composer update uvdesk/core-framework
composer update uvdesk/support-center-bundle
composer update uvdesk/mailbox-component
composer update uvdesk/automation-bundle
composer update uvdesk/extension-framework
```

than:

```
php bin/console d:s:u --force
php bin/console assets:install
```

I was not able to delete customer and need add manually the [fix here](https://github.com/uvdesk/core-framework/commit/442590fa5ba4e03c7b804a69cef361607e222406), why?

**Comment by Sanjaybhattwebkul** [comment]
@PeopleInside   
   All issues are fix in our  **new release V1.0.16** 
![CustomerList](https://user-images.githubusercontent.com/86394952/130446791-1f153f50-a597-41d7-9a7a-202dc884cb1c.png)
![Delete-Customer](https://user-images.githubusercontent.com/86394952/130446799-810570f8-4d88-4729-8711-8fa7de3b17c9.png)
![Ticket-status](https://user-images.githubusercontent.com/86394952/130446806-318155b3-1ad7-4638-83db-71bae2c56193.png)

**Comment by PeopleInside** [comment]
Thanks I will test, my customer are created by email not by the web interface as in your example.
V 1.0.15 was not working and I need manually fix, I will see in the new 1.0.16
